articles/protocols/saml/saml-configuration/troubleshoot/index.md

```diff
@@ -51,12 +51,6 @@ When troubleshooting, we recommend beginning by gathering information that helps
 4. What is the expected behavior? What is the behavior you're seeing?
 5. How far through the login sequence does the user get?
 
-## Troubleshooting Approaches
-
-* [SAML Troubleshooting When Auth0 is the Service Provider](/protocols/saml/saml-configuration/troubleshoot/auth0-as-sp)
-
-* [SAML Troubleshooting When Auth0 is the Identity Provider](/protocols/saml/saml-configuration/troubleshoot/auth0-as-idp)
-
 ## If the Issue Affects Only One (or Just a Few) Users
```

Comment:
I'm not quite sure about removing the "Troubleshooting approaches". Without them as the page content, this whole page has little substance, and the "Keep reading" list of articles is easily missed IMO.

Reply:
Added back in